### PR TITLE
1023007

### PR DESCRIPTION
**Pull request by a5135324**
1023007

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 
 int main() {
+	printf("1023007");
```
Comment (jrjang2) [inline whoami.c]: Leave an empty line before ```return 0;```.

**Comment by jrjang2** [comment]
Remove whoami.c~ .

**Comment by jrjang2** [comment]
Remove whoami.c~ .

**Comment by jrjang2** [comment]
Great. But there is a conflict now. Please solve this issue before doing pull request. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by a5135324** [comment]
I fork the project (jrjang2/lab) by clicking the button at the right-top corner. 
Then back to terminal and key in "git pull -- rebase", but it shows me "Current branch master is up to date."
Is there something wrong or something that I forgot to do? 
Thank you.

**Comment by jrjang2** [comment]
Great! Try to merge your commits to one before doing pull request.

Tips:
1. ```git rebase```
2. ```git reset --mixed```

**Comment by a5135324** [comment]
Done. ( Maybe XD )

**Comment by a5135324** [comment]
Done.

**Comment by jrjang2** [comment]
Good job. There is the new conflict now. Please fix it.

**Comment by a5135324** [comment]
Done.

**Comment by jrjang2** [comment]
Great!

**Pull request merged**